src/commands/georadiusbymember_ro.json

```diff
@@ -6,6 +6,12 @@
         "since": "3.2.10",
         "arity": -5,
         "function": "georadiusbymemberroCommand",
+        "history": [
+          [
+              "6.2.0",
+              "Added the `ANY` option for `COUNT`."
+          ]
+        ],
```

Comment:
```suggestion
    "history": [
          [
              "6.2.0",
              "Added the `ANY` option for `COUNT`."
          ],
          [
              "7.0.0", 
              "Added support for uppercase unit names."
          ]
        ],
```

Reply:
The above updates also need to be added to `georadius_ro.json`.

Reply:
LGTM